### DISCUSSION TITLE
Extra dependency "mysqlconnector" doesn't exist

### DISCUSSION THREAD

**Discussion by Huy-Ngo** [sqlalchemy/sqlalchemy#7012] [Usage Questions]
One of the extra dependency for SQLAlchemy is `mysqlconnector`, which can't be found on [PyPI](https://pypi.org/project/mysqlconnector/). Is this a mistake and it means to be [`mysql-connector`](https://pypi.org/project/mysql-connector/) instead? Or the package can be found somewhere else?

**Comment by zzzeek** [comment]
driver is documented at https://docs.sqlalchemy.org/en/14/dialects/mysql.html#module-sqlalchemy.dialects.mysql.mysqlconnector will update the setup.cfg

**Reply by zzzeek** [reply]
fixed in 9ca7be23af16e3cf958

**Comment by Huy-Ngo** [comment]
Thanks for the fix
